src/lib.rs

```diff
@@ -964,6 +965,14 @@ impl<F: Read + Write + Seek> CompoundFile<F> {
     }
 }
 
+impl<F: fmt::Debug> fmt::Debug for CompoundFile<F> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_tuple("CompoundFile")
+            .field(self.minialloc().inner().inner().inner().inner())
```

Comment:
Once the definitions of the various `inner()` methods are changed, then this can just become `.field(self.minialloc().inner())`.